gems/ruby-saml/CVE-2016-5697.yml

```diff
@@ -0,0 +1,14 @@
+---
+gem: ruby-saml
+cve: 2016-5697
+url: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-5697
```

Comment:
These URLs are fairly useless, as they almost never update them nowadays, and we actually will link to them by default due to the CVE assignment. We want this URL to be something a developer or security engineer could use to see immediate information about the issue. My recommendation would be one of the following (in decreasing priority):
* https://github.com/onelogin/ruby-saml/commit/a571f52171e6bfd87db59822d1d9e8c38fb3b995
* https://github.com/onelogin/ruby-saml/releases/tag/v1.3.0
* http://seclists.org/oss-sec/2016/q2/597 or https://groups.google.com/forum/#!topic/rubysec-announce/vqHizIdb07M